### PR TITLE
add jsroot component (#223)

### LINKED ISSUE #223: integrate dummy react component with JSROOT integrated
This component should display a dummy JSROOT plot in the Results tab.
The data should be specified as a list of numbers, not as the root file.

For inspiration https://github.com/APTG/dedx_web

### PR DESCRIPTION
**Pull request by ostatni5**
close #223 

obraz

### CODE REVIEW

File: src/JsRoot/JsRoot.tsx [hunk 1]
```diff
@@ -2,44 +2,49 @@ import Loader from "../Loader/Loader";
 import makeAsyncScriptLoader from "react-async-script";
 import React, { useEffect, useRef, useState } from 'react';
 
-const URL = `https://root.cern.ch/js/latest/scripts/JSRoot.core.js`;
-const globalName = "JSROOT";
+const urlJSROOT = 'https://root.cern.ch/js/latest/scripts/JSRoot.core.js';
+const keyJSROOT = 'JSROOT';
 
+declare global {
```
Comment (grzanka) [inline src/JsRoot/JsRoot.tsx:8]: Why this is `global` ?

File: src/JsRoot/JsRoot.tsx [hunk 2]
```diff
@@ -2,44 +2,49 @@ import Loader from "../Loader/Loader";
...
-        JSROOT.openFile("./files/hsimple.root")
-            .then((file : any) => file.readObject("hpxpy;1"))
-            .then((obj : any) => {
-                setObj(obj) ;
-                setScriptLoaded(true);
-            });
+        const { JSROOT } = window;
+
+        // create example graph
+        let nbinsx = 20;
```
Comment (grzanka) [inline src/JsRoot/JsRoot.tsx:23]:
```suggestion
        const nbinsx = 20;
```
I agree with deepsource

**Review by grzanka — COMMENTED** [review]
see comments

File: src/JsRoot/JsRoot.tsx (hunk 1 above)
Comment (ostatni5) [reply src/JsRoot/JsRoot.tsx:8]: https://stackoverflow.com/a/12709880

**Review by Derstilon — APPROVED** [review]

**Comment by grzanka** [comment]
@ostatni5 fix conflicts:

image

**Comment by grzanka** [comment]
Still, the JSROOT panel is not visible in the Results tab. Moreover, some interesting warnings are seen in developer console, indicating problem with canvas size:

image

**Review by grzanka — DISMISSED** [review]
works nice

**Review by grzanka — APPROVED** [review]
still good

**Review by Derstilon — APPROVED** [review]
Works fine

**Pull request merged**